csrc/net/trt/CMakeLists.txt

```diff
@@ -2,6 +2,11 @@
 cmake_minimum_required(VERSION 3.14)
 project(mmdeploy_trt_net)
 
+if (${CMAKE_VERSION} VERSION_GREATER_EQUAL "3.18.0")
```

Comment:
Can we do that in `cuda.cmake`? Otherwise, it still warns `CMAKE_CUDA_ARCHITECTURES` when build the target `mmdeploy_tensorrt_ops`.